### PR TITLE
[ Application ] VGG using learning rate decay

### PR DESCRIPTION
**Pull request by jijoongmoon**
Change to use learning rate decay

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

**Review by kparichay — APPROVED** [review]
LGTM
@jijoongmoon I see that this doesn't make any significant changes for the tf stabilized training (except what we discussed about softmax). So, if these are good, this can be merged.
I will look more closely as to why tf model isn't training

**Review by zhoonit — APPROVED** [review]
LGTM!

**Comment by kparichay** [comment]
@jijoongmoon Please rebase.
